3ds Max/Max2Babylon/Exporter/BabylonExporter.Material.cs

```diff
@@ -1,509 +1,605 @@
+                                    else
+                                    {
+                                        babylonMaterial.occlusionTexture = ormTexture;
+                                    }
+                                }
+                                else
+                                {
+                                    babylonMaterial.occlusionTexture = ExportPBRTexture(materialNode, 6, babylonScene);
+                                }
+                            }
+                        }
+                        else if (ambientOcclusionTexmap != null)
+                        {
+                            // Simply export occlusion texture
+                            RaiseVerbose("Simply export occlusion texture", 2);
+                            babylonMaterial.occlusionTexture = ExportTexture(ambientOcclusionTexmap, babylonScene);
+                        }
+                    }
+                    if (ambientOcclusionTexmap != null && !exportParameters.mergeAOwithMR && babylonMaterial.occlusionTexture == null)
+                    {
+                        RaiseVerbose("Exporting occlusion texture without merging with metallic roughness", 2);
+                        babylonMaterial.occlusionTexture = ExportTexture(ambientOcclusionTexmap, babylonScene);
+                    }
+
+                    var normalMapAmount = propertyContainer.GetFloatProperty(91);
+                    babylonMaterial.normalTexture = ExportPBRTexture(materialNode, 30, babylonScene, normalMapAmount);
+
+                    babylonMaterial.emissiveTexture = ExportPBRTexture(materialNode, 17, babylonScene);
+                }
+
+                if (isTransparencyModeFromBabylonAttributes == false && (babylonMaterial.alpha != 1.0f || (babylonMaterial.baseTexture != null && babylonMaterial.baseTexture.hasAlpha)))
+                {
+                    babylonMaterial.transparencyMode = (int)BabylonPBRMetallicRoughnessMaterial.TransparencyMode.ALPHABLEND;
+                }
+
+                if (babylonMaterial.emissiveTexture != null)
+                {
+                    babylonMaterial.emissive = new[] { 1.0f, 1.0f, 1.0f };
+                }
+
+                if (babylonMaterial.metallicRoughnessTexture != null)
+                {
+                    babylonMaterial.metallic = 1.0f;
+                    babylonMaterial.roughness = 1.0f;
+                }
+
+                if (babylonMaterial.transparencyMode == (int)BabylonPBRMetallicRoughnessMaterial.TransparencyMode.ALPHATEST)
+                {
+                    // Set the alphaCutOff value explicitely to avoid different interpretations on different engines
+                    // Use the glTF default value rather than the babylon one
+                    babylonMaterial.alphaCutOff = 0.5f;
+                }
+
+                // Add babylon attributes
+                AddPhysicalBabylonAttributes(materialNode.MaterialName, babylonMaterial);

```

Comment:
Similarly to the Maya exporter, after testing the workflow, requiring a "dryrun" to add custom attributes feels like a suboptimal experience. Can we look into possible ways to hook into the slate editor and add attributes on node creation? Or, at possibly expose a UI or context menu control that does this?